src/librustc/middle/cstore.rs

```diff
@@ -40,6 +40,8 @@ use rustc_data_structures::sync::{self, MetadataRef, Lrc};
 
 pub use self::NativeLibraryKind::*;
 
+pub const METADATA_FILENAME: &str = "rust.metadata.bin";
```

Comment:
Why have we moved this? It might be useful in the commit message to motivate the change.

Reply:
My bad, this wasn't supposed to show up in the diff. This line is actually part of my refactoring branch (which we've already merged). I originally worked on a single branch, and it seems I messed up when splitting it in two. It's fixed now, all the commits should be relevant to this PR. Sorry about that!